### PR TITLE
New build from my ARM mac 🙌

### PR DESCRIPTION
**Pull request by juliasilge**
I have gotten it to work, finally 🥴

### CODE REVIEW

**Comment by juliasilge** [comment]
I scrolled through and looked in detail at all the figures that changed; they look good to me, maybe with the exception of the lime figure which is a little "meh" now. 🤷‍♀️

File: docs/08_dl_dnn.md [hunk 1]
```diff
@@ -1241,10 +1239,10 @@ cv_fitted %>%
 #> # A tibble: 4 × 4
 #>   .metric      mean     n  std_err
 #>   <chr>       <dbl> <int>    <dbl>
-#> 1 accuracy    0.819     5 0.000523
```
Comment (EmilHvitfeldt) [inline docs/08_dl_dnn.md:1244]: Is it expected that we are getting different results here?

Comment (juliasilge) [reply docs/08_dl_dnn.md:1244]: It's the same as Ch 9 and 10, right? That the seeds aren't really reproducible into the Python process?

File: docs/07_ml_classification.md [hunk 2]
```diff
@@ -349,8 +349,8 @@ nb_rs_metrics
 #> # A tibble: 2 × 6
 #>   .metric  .estimator  mean     n std_err .config
```
Comment (EmilHvitfeldt) [inline docs/07_ml_classification.md:350]: Sorry I was linking to the wrong output. This one are results from a `naivebayes` model. No python

File: docs/08_dl_dnn.md (hunk 1 above)
Comment (EmilHvitfeldt) [reply docs/08_dl_dnn.md:1244]: I made a mistake. Please refer to the other comment

File: docs/07_ml_classification.md (hunk 2 above)
Comment (juliasilge) [reply docs/07_ml_classification.md:350]: Ah yes, I thought this was interesting. If I run this interactively, I do get the green (new) results. Do you still get the old results with this set of packages?

Comment (EmilHvitfeldt) [reply docs/07_ml_classification.md:350]: Hmm. I'm getting the old versions

Screen Shot 2021-11-12 at 9 16 33 AM

Comment (EmilHvitfeldt) [reply docs/07_ml_classification.md:350]: Doo you differ on anything here:

```r
─ Session info ──────────────────────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 4.1.0 (2021-05-18)
 os       macOS Big Sur 11.3.1        
 system   x86_64, darwin17.0          
 ui       RStudio                     
 language (EN)                        
 collate  en_US.UTF-8                 
 ctype    en_US.UTF-8                 
 tz       America/Los_Angeles         
 date     2021-11-12                  

─ Packages ──────────────────────────────────────────────────────────────────────────────────
 ! package      * version    date       lib source                        
 P assertthat     0.2.1      2019-03-21 [?] CRAN (R 4.1.0)                
 P backports      1.2.1      2020-12-09 [?] CRAN (R 4.1.0)                
 P bit            4.0.4      2020-08-04 [?] CRAN (R 4.1.0)                
 P bit64          4.0.5      2020-08-30 [?] CRAN (R 4.1.0)                
 P bookdown       0.24       2021-09-02 [?] CRAN (R 4.1.0)                
 P broom        * 0.7.9      2021-07-27 [?] CRAN (R 4.1.0)                
 P cachem         1.0.6      2021-08-19 [?] CRAN (R 4.1.0)                
 P callr          3.7.0      2021-04-20 [?] CRAN (R 4.1.0)                
 P cellranger     1.1.0      2016-07-27 [?] CRAN (R 4.1.0)                
 P class          7.3-19     2021-05-03 [?] CRAN (R 4.1.0)                
 P cli            3.0.1      2021-07-17 [?] CRAN (R 4.1.0)                
 P codetools      0.2-18     2020-11-04 [?] CRAN (R 4.1.0)                
 P colorspace     2.0-2      2021-06-24 [?] CRAN (R 4.1.0)                
 P conflicted     1.0.4      2019-06-21 [?] CRAN (R 4.1.0)                
 P crayon         1.4.1      2021-02-08 [?] CRAN (R 4.1.0)                
 P data.table     1.14.2     2021-09-27 [?] CRAN (R 4.1.0)                
 P DBI            1.1.1      2021-01-15 [?] CRAN (R 4.1.0)                
 P dbplyr         2.1.1      2021-04-06 [?] CRAN (R 4.1.0)                
 P desc           1.4.0      2021-09-28 [?] CRAN (R 4.1.0)                
 P devtools       2.4.2      2021-06-07 [?] CRAN (R 4.1.0)                
 P dials        * 0.0.10     2021-09-10 [?] CRAN (R 4.1.0)                
 P DiceDesign     1.9        2021-02-13 [?] CRAN (R 4.1.0)                
 P digest         0.6.28     2021-09-23 [?] CRAN (R 4.1.0)                
 P discrim      * 0.1.3      2021-07-21 [?] CRAN (R 4.1.0)                
 P dplyr        * 1.0.7      2021-06-18 [?] CRAN (R 4.1.0)                
 P ellipsis       0.3.2      2021-04-29 [?] CRAN (R 4.1.0)                
 P evaluate       0.14       2019-05-28 [?] CRAN (R 4.1.0)                
 P fansi          0.5.0      2021-05-25 [?] CRAN (R 4.1.0)                
 P farver         2.1.0      2021-02-28 [?] CRAN (R 4.1.0)                
 P fastmap        1.1.0      2021-01-25 [?] CRAN (R 4.1.0)                
 P fastmatch      1.1-3      2021-07-23 [?] CRAN (R 4.1.0)                
 P forcats      * 0.5.1      2021-01-27 [?] CRAN (R 4.1.0)                
 P foreach        1.5.1      2020-10-15 [?] CRAN (R 4.1.0)                
 P fs             1.5.0      2020-07-31 [?] CRAN (R 4.1.0)                
 P furrr          0.2.3      2021-06-25 [?] CRAN (R 4.1.0)                
 P future         1.22.1     2021-08-25 [?] CRAN (R 4.1.0)                
 P future.apply   1.8.1      2021-08-10 [?] CRAN (R 4.1.0)                
 P generics       0.1.0      2020-10-31 [?] CRAN (R 4.1.0)                
 P ggplot2      * 3.3.5      2021-06-25 [?] CRAN (R 4.1.0)                
 P globals        0.14.0     2020-11-22 [?] CRAN (R 4.1.0)                
 P glue           1.4.2      2020-08-27 [?] CRAN (R 4.1.0)                
 P gower          0.2.2      2020-06-23 [?] CRAN (R 4.1.0)                
 P GPfit          1.0-8      2019-02-08 [?] CRAN (R 4.1.0)                
 P gridExtra      2.3        2017-09-09 [?] CRAN (R 4.1.0)                
 P gtable         0.3.0      2019-03-25 [?] CRAN (R 4.1.0)                
 P hardhat        0.1.6      2021-07-14 [?] CRAN (R 4.1.0)                
 P haven          2.4.3      2021-08-04 [?] CRAN (R 4.1.0)                
 P hms            1.1.1      2021-09-26 [?] CRAN (R 4.1.0)                
 P htmltools    * 0.5.2      2021-08-25 [?] CRAN (R 4.1.0)                
 P htmlwidgets    1.5.4      2021-09-08 [?] CRAN (R 4.1.0)                
 P httr           1.4.2      2020-07-20 [?] CRAN (R 4.1.0)                
 P infer        * 1.0.0      2021-08-13 [?] CRAN (R 4.1.0)                
 P ipred          0.9-12     2021-09-15 [?] CRAN (R 4.1.0)                
 P iterators      1.0.13     2020-10-15 [?] CRAN (R 4.1.0)                
 P jsonlite       1.7.2      2020-12-09 [?] CRAN (R 4.1.0)                
 P knitr        * 1.36       2021-09-29 [?] CRAN (R 4.1.0)                
 P lattice        0.20-45    2021-09-22 [?] CRAN (R 4.1.0)                
 P lava           1.6.10     2021-09-02 [?] CRAN (R 4.1.0)                
 P lhs            1.1.3      2021-09-08 [?] CRAN (R 4.1.0)                
 P lifecycle      1.0.1      2021-09-24 [?] CRAN (R 4.1.0)                
 P listenv        0.8.0      2019-12-05 [?] CRAN (R 4.1.0)                
 P lubridate      1.8.0      2021-10-07 [?] CRAN (R 4.1.0)                
 P magrittr       2.0.1      2020-11-17 [?] CRAN (R 4.1.0)                
 P MASS           7.3-54     2021-05-03 [?] CRAN (R 4.1.0)                
 P Matrix         1.3-4      2021-06-01 [?] CRAN (R 4.1.0)                
 P memoise        2.0.0      2021-01-26 [?] CRAN (R 4.1.0)                
 P modeldata    * 0.1.1      2021-07-14 [?] CRAN (R 4.1.0)                
 P modelr         0.1.8      2020-05-19 [?] CRAN (R 4.1.0)                
 P munsell        0.5.0      2018-06-12 [?] CRAN (R 4.1.0)                
 P naivebayes   * 0.9.7      2020-03-08 [?] CRAN (R 4.1.0)                
 P nnet           7.3-16     2021-05-03 [?] CRAN (R 4.1.0)                
 P parallelly     1.28.1     2021-09-09 [?] CRAN (R 4.1.0)                
 P parsnip      * 0.1.7      2021-07-21 [?] CRAN (R 4.1.0)                
 P pillar         1.6.4      2021-10-18 [?] CRAN (R 4.1.0)                
 P pkgbuild       1.2.0      2020-12-15 [?] CRAN (R 4.1.0)                
 P pkgconfig      2.0.3      2019-09-22 [?] CRAN (R 4.1.0)                
 P pkgload        1.2.3      2021-10-13 [?] CRAN (R 4.1.0)                
 P plyr           1.8.6      2020-03-03 [?] CRAN (R 4.1.0)                
 P png            0.1-7      2013-12-03 [?] CRAN (R 4.1.0)                
 P prettyunits    1.1.1      2020-01-24 [?] CRAN (R 4.1.0)                
 P prismatic      1.1.0      2021-10-17 [?] CRAN (R 4.1.0)                
 P pROC           1.18.0     2021-09-03 [?] CRAN (R 4.1.0)                
 P processx       3.5.2      2021-04-30 [?] CRAN (R 4.1.0)                
 P prodlim        2019.11.13 2019-11-17 [?] CRAN (R 4.1.0)                
 P ps             1.6.0      2021-02-28 [?] CRAN (R 4.1.0)                
 P purrr        * 0.3.4      2020-04-17 [?] CRAN (R 4.1.0)                
 P quanteda     * 3.1.0      2021-08-17 [?] CRAN (R 4.1.0)                
 P R6             2.5.1      2021-08-19 [?] CRAN (R 4.1.0)                
 P Rcpp           1.0.7      2021-07-07 [?] CRAN (R 4.1.0)                
 P RcppParallel   5.1.4      2021-05-04 [?] CRAN (R 4.1.0)                
 P readr        * 2.0.2      2021-09-27 [?] CRAN (R 4.1.0)                
 P readxl         1.3.1      2019-03-13 [?] CRAN (R 4.1.0)                
 P recipes      * 0.1.17     2021-09-27 [?] CRAN (R 4.1.0)                
 P remotes        2.4.1      2021-09-29 [?] CRAN (R 4.1.0)                
   renv           0.14.0     2021-07-21 [1] CRAN (R 4.1.0)                
 P reprex         2.0.1      2021-08-05 [?] CRAN (R 4.1.0)                
 P reticulate     1.22       2021-09-17 [?] CRAN (R 4.1.0)                
 P rlang        * 0.4.12     2021-10-18 [?] CRAN (R 4.1.0)                
 P rmarkdown      2.11       2021-09-14 [?] CRAN (R 4.1.0)                
 P rpart          4.1-15     2019-04-12 [?] CRAN (R 4.1.0)                
 P rprojroot      2.0.2      2020-11-15 [?] CRAN (R 4.1.0)                
 P rsample      * 0.1.0      2021-05-08 [?] CRAN (R 4.1.0)                
 P rstudioapi     0.13       2020-11-12 [?] CRAN (R 4.1.0)                
 P rvest          1.0.2      2021-10-16 [?] CRAN (R 4.1.0)                
 P scales       * 1.1.1      2020-05-11 [?] CRAN (R 4.1.0)                
 P sessioninfo    1.1.1      2018-11-05 [?] CRAN (R 4.1.0)                
 P SnowballC      0.7.0      2020-04-01 [?] CRAN (R 4.1.0)                
 P spacyr         1.2.1      2020-03-04 [?] CRAN (R 4.1.0)                
 P stopwords      2.2        2021-02-10 [?] CRAN (R 4.1.0)                
 P stringi        1.7.5      2021-10-04 [?] CRAN (R 4.1.0)                
 P stringr      * 1.4.0      2019-02-10 [?] CRAN (R 4.1.0)                
 P survival       3.2-13     2021-08-24 [?] CRAN (R 4.1.0)                
 P testthat       3.1.0      2021-10-04 [?] CRAN (R 4.1.0)                
 P textrecipes  * 0.4.1      2021-07-11 [?] CRAN (R 4.1.0)                
 P tibble       * 3.1.5      2021-09-30 [?] CRAN (R 4.1.0)                
 P tidymodels   * 0.1.4      2021-10-01 [?] CRAN (R 4.1.0)                
 P tidyr        * 1.1.4      2021-09-27 [?] CRAN (R 4.1.0)                
 P tidyselect     1.1.1      2021-04-30 [?] CRAN (R 4.1.0)                
 P tidyverse    * 1.3.1      2021-04-15 [?] CRAN (R 4.1.0)                
 P timeDate       3043.102   2018-02-21 [?] CRAN (R 4.1.0)                
 P tokenizers     0.2.1      2018-03-29 [?] CRAN (R 4.1.0)                
 P tune         * 0.1.6      2021-07-21 [?] CRAN (R 4.1.0)                
 P tzdb           0.1.2      2021-07-20 [?] CRAN (R 4.1.0)                
 P usethis        2.1.1      2021-10-21 [?] Github (r-lib/usethis@e05dd0f)
 P utf8           1.2.2      2021-07-24 [?] CRAN (R 4.1.0)                
 P vctrs        * 0.3.8      2021-04-29 [?] CRAN (R 4.1.0)                
 P viridis        0.6.2      2021-10-13 [?] CRAN (R 4.1.0)                
 P viridisLite    0.4.0      2021-04-13 [?] CRAN (R 4.1.0)                
 P vroom          1.5.5      2021-09-14 [?] CRAN (R 4.1.0)                
 P withr          2.4.2      2021-04-18 [?] CRAN (R 4.1.0)                
 P workflows    * 0.2.4      2021-10-12 [?] CRAN (R 4.1.0)                
 P workflowsets * 0.1.0      2021-07-22 [?] CRAN (R 4.1.0)                
 P xfun           0.27       2021-10-18 [?] CRAN (R 4.1.0)                
 P xml2           1.3.2      2020-04-23 [?] CRAN (R 4.1.0)                
 P yaml           2.2.1      2020-02-01 [?] CRAN (R 4.1.0)                
 P yardstick    * 0.0.8      2021-03-28 [?] CRAN (R 4.1.0)                
```

Comment (juliasilge) [reply docs/07_ml_classification.md:350]: I just ran once again, and here is what I still have:
```
> nb_rs_metrics
# A tibble: 2 × 6
  .metric  .estimator  mean     n std_err .config             
  <chr>    <chr>      <dbl> <int>   <dbl> <chr>               
1 accuracy binary     0.802    10 0.00446 Preprocessor1_Model1
2 roc_auc  binary     0.881    10 0.00168 Preprocessor1_Model1
```

One thought: we aren't setting a seed before the NB model?

## Session info:

```
> sessioninfo::session_info()
─ Session info ───────────────────────────────────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 4.1.1 (2021-08-10)
 os       macOS Big Sur 11.6          
 system   aarch64, darwin20           
 ui       RStudio                     
 language (EN)                        
 collate  en_US.UTF-8                 
 ctype    en_US.UTF-8                 
 tz       America/Denver              
 date     2021-11-16                  

─ Packages ───────────────────────────────────────────────────────────────────────────────────────────────
 ! package      * version    date       lib source        
 P assertthat     0.2.1      2019-03-21 [?] CRAN (R 4.1.0)
 P backports      1.2.1      2020-12-09 [?] CRAN (R 4.1.0)
 P bit            4.0.4      2020-08-04 [?] CRAN (R 4.1.1)
 P bit64          4.0.5      2020-08-30 [?] CRAN (R 4.1.0)
 P bookdown       0.24       2021-09-02 [?] CRAN (R 4.1.1)
 P broom        * 0.7.9      2021-07-27 [?] CRAN (R 4.1.0)
 P cachem         1.0.6      2021-08-19 [?] CRAN (R 4.1.1)
 P cellranger     1.1.0      2016-07-27 [?] CRAN (R 4.1.0)
 P class          7.3-19     2021-05-03 [?] CRAN (R 4.1.1)
 P cli            3.0.1      2021-07-17 [?] CRAN (R 4.1.0)
 P codetools      0.2-18     2020-11-04 [?] CRAN (R 4.1.1)
 P colorspace     2.0-2      2021-06-24 [?] CRAN (R 4.1.1)
 P conflicted     1.0.4      2019-06-21 [?] CRAN (R 4.1.1)
 P crayon         1.4.1      2021-02-08 [?] CRAN (R 4.1.0)
 P data.table     1.14.2     2021-09-27 [?] CRAN (R 4.1.1)
 P DBI            1.1.1      2021-01-15 [?] CRAN (R 4.1.0)
 P dbplyr         2.1.1      2021-04-06 [?] CRAN (R 4.1.0)
 P desc           1.4.0      2021-09-28 [?] CRAN (R 4.1.1)
 P dials        * 0.0.10     2021-09-10 [?] CRAN (R 4.1.1)
 P DiceDesign     1.9        2021-02-13 [?] CRAN (R 4.1.0)
 P digest         0.6.28     2021-09-23 [?] CRAN (R 4.1.1)
 P discrim      * 0.1.3      2021-07-21 [?] CRAN (R 4.1.0)
 P dplyr        * 1.0.7      2021-06-18 [?] CRAN (R 4.1.0)
 P ellipsis       0.3.2      2021-04-29 [?] CRAN (R 4.1.0)
 P evaluate       0.14       2019-05-28 [?] CRAN (R 4.1.0)
 P fansi          0.5.0      2021-05-25 [?] CRAN (R 4.1.0)
 P fastmap        1.1.0      2021-01-25 [?] CRAN (R 4.1.0)
 P fastmatch      1.1-3      2021-07-23 [?] CRAN (R 4.1.0)
 P forcats      * 0.5.1      2021-01-27 [?] CRAN (R 4.1.1)
 P foreach        1.5.1      2020-10-15 [?] CRAN (R 4.1.0)
 P fs             1.5.0      2020-07-31 [?] CRAN (R 4.1.0)
 P furrr          0.2.3      2021-06-25 [?] CRAN (R 4.1.0)
 P future         1.22.1     2021-08-25 [?] CRAN (R 4.1.1)
 P future.apply   1.8.1      2021-08-10 [?] CRAN (R 4.1.1)
 P generics       0.1.0      2020-10-31 [?] CRAN (R 4.1.0)
 P ggplot2      * 3.3.5      2021-06-25 [?] CRAN (R 4.1.1)
 P globals        0.14.0     2020-11-22 [?] CRAN (R 4.1.0)
 P glue           1.4.2      2020-08-27 [?] CRAN (R 4.1.0)
 P gower          0.2.2      2020-06-23 [?] CRAN (R 4.1.0)
 P GPfit          1.0-8      2019-02-08 [?] CRAN (R 4.1.0)
 P gridExtra      2.3        2017-09-09 [?] CRAN (R 4.1.1)
 P gtable         0.3.0      2019-03-25 [?] CRAN (R 4.1.1)
 P hardhat        0.1.6      2021-07-14 [?] CRAN (R 4.1.0)
 P haven          2.4.3      2021-08-04 [?] CRAN (R 4.1.1)
 P hms            1.1.1      2021-09-26 [?] CRAN (R 4.1.1)
 P htmltools    * 0.5.2      2021-08-25 [?] CRAN (R 4.1.1)
 P httr           1.4.2      2020-07-20 [?] CRAN (R 4.1.0)
 P infer        * 1.0.0      2021-08-13 [?] CRAN (R 4.1.1)
 P ipred          0.9-12     2021-09-15 [?] CRAN (R 4.1.1)
 P iterators      1.0.13     2020-10-15 [?] CRAN (R 4.1.0)
 P jsonlite       1.7.2      2020-12-09 [?] CRAN (R 4.1.0)
 P knitr        * 1.36       2021-09-29 [?] CRAN (R 4.1.1)
 P lattice        0.20-45    2021-09-22 [?] CRAN (R 4.1.1)
 P lava           1.6.10     2021-09-02 [?] CRAN (R 4.1.1)
 P lhs            1.1.3      2021-09-08 [?] CRAN (R 4.1.1)
 P lifecycle      1.0.1      2021-09-24 [?] CRAN (R 4.1.1)
 P listenv        0.8.0      2019-12-05 [?] CRAN (R 4.1.0)
 P lubridate      1.8.0      2021-10-07 [?] CRAN (R 4.1.1)
 P magrittr       2.0.1      2020-11-17 [?] CRAN (R 4.1.0)
 P MASS           7.3-54     2021-05-03 [?] CRAN (R 4.1.1)
 P Matrix         1.3-4      2021-06-01 [?] CRAN (R 4.1.1)
 P modeldata    * 0.1.1      2021-07-14 [?] CRAN (R 4.1.0)
 P modelr         0.1.8      2020-05-19 [?] CRAN (R 4.1.0)
 P munsell        0.5.0      2018-06-12 [?] CRAN (R 4.1.0)
 P naivebayes   * 0.9.7      2020-03-08 [?] CRAN (R 4.1.0)
 P nnet           7.3-16     2021-05-03 [?] CRAN (R 4.1.1)
 P parallelly     1.28.1     2021-09-09 [?] CRAN (R 4.1.1)
 P parsnip      * 0.1.7      2021-07-21 [?] CRAN (R 4.1.0)
 P pillar         1.6.4      2021-10-18 [?] CRAN (R 4.1.1)
 P pkgconfig      2.0.3      2019-09-22 [?] CRAN (R 4.1.0)
 P plyr           1.8.6      2020-03-03 [?] CRAN (R 4.1.0)
 P pROC           1.18.0     2021-09-03 [?] CRAN (R 4.1.1)
 P prodlim        2019.11.13 2019-11-17 [?] CRAN (R 4.1.0)
 P purrr        * 0.3.4      2020-04-17 [?] CRAN (R 4.1.0)
 P quanteda     * 3.1.0      2021-08-17 [?] CRAN (R 4.1.1)
 P R6             2.5.1      2021-08-19 [?] CRAN (R 4.1.1)
 P Rcpp           1.0.7      2021-07-07 [?] CRAN (R 4.1.0)
 P RcppParallel   5.1.4      2021-05-04 [?] CRAN (R 4.1.0)
 P readr        * 2.0.2      2021-09-27 [?] CRAN (R 4.1.1)
 P readxl         1.3.1      2019-03-13 [?] CRAN (R 4.1.0)
 P recipes      * 0.1.17     2021-09-27 [?] CRAN (R 4.1.1)
   renv           0.14.0     2021-07-21 [1] CRAN (R 4.1.0)
 P reprex         2.0.1      2021-08-05 [?] CRAN (R 4.1.1)
 P rlang        * 0.4.12     2021-10-18 [?] CRAN (R 4.1.1)
 P rmarkdown      2.11       2021-09-14 [?] CRAN (R 4.1.1)
 P rpart          4.1-15     2019-04-12 [?] CRAN (R 4.1.1)
 P rprojroot      2.0.2      2020-11-15 [?] CRAN (R 4.1.0)
 P rsample      * 0.1.0      2021-05-08 [?] CRAN (R 4.1.1)
 P rstudioapi     0.13       2020-11-12 [?] CRAN (R 4.1.0)
 P rvest          1.0.2      2021-10-16 [?] CRAN (R 4.1.1)
 P scales       * 1.1.1      2020-05-11 [?] CRAN (R 4.1.0)
 P sessioninfo    1.1.1      2018-11-05 [?] CRAN (R 4.1.0)
 P SnowballC      0.7.0      2020-04-01 [?] CRAN (R 4.1.0)
 P spacyr         1.2.1      2020-03-04 [?] CRAN (R 4.1.0)
 P stopwords      2.2        2021-02-10 [?] CRAN (R 4.1.0)
 P stringi        1.7.5      2021-10-04 [?] CRAN (R 4.1.1)
 P stringr      * 1.4.0      2019-02-10 [?] CRAN (R 4.1.1)
 P survival       3.2-13     2021-08-24 [?] CRAN (R 4.1.1)
 P textrecipes  * 0.4.1      2021-07-11 [?] CRAN (R 4.1.0)
 P tibble       * 3.1.5      2021-09-30 [?] CRAN (R 4.1.1)
 P tidymodels   * 0.1.4      2021-10-01 [?] CRAN (R 4.1.1)
 P tidyr        * 1.1.4      2021-09-27 [?] CRAN (R 4.1.1)
 P tidyselect     1.1.1      2021-04-30 [?] CRAN (R 4.1.0)
 P tidyverse    * 1.3.1      2021-04-15 [?] CRAN (R 4.1.0)
 P timeDate       3043.102   2018-02-21 [?] CRAN (R 4.1.0)
 P tokenizers     0.2.1      2018-03-29 [?] CRAN (R 4.1.0)
 P tune         * 0.1.6      2021-07-21 [?] CRAN (R 4.1.0)
 P tzdb           0.1.2      2021-07-20 [?] CRAN (R 4.1.0)
 P utf8           1.2.2      2021-07-24 [?] CRAN (R 4.1.0)
 P vctrs        * 0.3.8      2021-04-29 [?] CRAN (R 4.1.0)
 P viridis        0.6.2      2021-10-13 [?] CRAN (R 4.1.1)
 P viridisLite    0.4.0      2021-04-13 [?] CRAN (R 4.1.0)
 P vroom          1.5.5      2021-09-14 [?] CRAN (R 4.1.1)
 P withr          2.4.2      2021-04-18 [?] CRAN (R 4.1.1)
 P workflows    * 0.2.4      2021-10-12 [?] CRAN (R 4.1.1)
 P workflowsets * 0.1.0      2021-07-22 [?] CRAN (R 4.1.1)
 P xfun           0.27       2021-10-18 [?] CRAN (R 4.1.1)
 P xml2           1.3.2      2020-04-23 [?] CRAN (R 4.1.0)
 P yaml           2.2.1      2020-02-01 [?] CRAN (R 4.1.0)
 P yardstick    * 0.0.8      2021-03-28 [?] CRAN (R 4.1.0)

[1] /Users/juliasilge/Work/tidytext/smltar/renv/library/R-4.1/aarch64-apple-darwin20
[2] /private/var/folders/hv/hzsmmyk9393_m7q3nscx1slc0000gn/T/RtmpW8LiRi/renv-system-library
```

Did you run that `session_info()` outside of renv? Most of the package versions look the same but there are some differences in what packages are actually _there_. We do have different OSes and different versions of R.

Comment (EmilHvitfeldt) [reply docs/07_ml_classification.md:350]: I moved to 4.1.1. Still the same result from me 😞  I restarted my renv and still nothing.

Comment (juliasilge) [reply docs/07_ml_classification.md:350]: I tried setting a seed right before `fit_resamples()` and it did not change my result.

I think the best thing to do at this point is probably to open an issue here on this repo and track this difference between ARM and Intel architecture; maybe we will see what else like this crops up as more people get ARM chips.

Comment (EmilHvitfeldt) [reply docs/07_ml_classification.md:350]: I'm happy with that. I'm getting a ARM one soon, so I can try to compare locally. Merge when like 😄

**Pull request merged**